### ISSUE TITLE
[LINT_BUG]: shiny$tags$br() isn't treated as shiny usage

### ISSUE THREAD

**Issue by TymekDev** [Appsilon/box.linters#148]
### box.linters version

v0.10.5

### Sample source code to lint

```r
box::use(
  shiny,
)

shiny$tags$br()
```

### Lint command used

```r
lintr::lint(
  "box::use(
  shiny,
)

shiny$tags$br()",
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)
```

### Lint result

```
<text>:2:3: warning: [box_unused_att_pkg_linter] Attached package unused.
  shiny,
  ^~~~~
```

### Expected result

I expect `shiny` to not be linted - its `tags` list is used.
